src/PinAssignment/PackagePinsModel.cpp

```diff
@@ -22,6 +22,18 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 namespace FOEDAG {
 
-PackagePinsModel::PackagePinsModel() {}
+PackagePinsModel::PackagePinsModel(QObject *parent) : QObject(parent) {}
+
+QStringList PackagePinsModel::headerList() const {
+  return {"Name", "Available",  "Ports",     "Ref clock",   "Bank",
```

Comment:
Shouldn't we generate this from the CSV file or create a mapping between the two?

Reply:
That wasn't an idea but we can have a  discussion on that